src/pages/api/featured-products.ts

```diff
@@ -29,7 +29,7 @@ export default async function handler(
 ) {
   setTimeout(() => {
     res.send(products);
-  }, 1500);
+  }, 750);
```

Comment:
I added long loading to make him start thinking about adding a loader. He might not have time to even get to this part of website before it gets loaded.

Reply:
But I changed the `FeaturedProducts` to be purely client side, so now when we open the page there are no products and they showed up after 1,5 seconds, so I just decided to make it shorter, as looking at a blank page for that long was kinda annoying :D I suggest running it locally to see what I mean and we can easily revert this change if we want :)